fill.go

```diff
@@ -101,7 +100,7 @@ func fillListeners(impl any, get func(name string) (net.Listener, string, error)
 
 		// The listener's name is the field name, unless a tag is present.
 		name := t.Name
-		if tag := t.Tag.Get("weaver"); tag != "" {
+		if tag, ok := t.Tag.Lookup("weaver"); ok {
```

Comment:
This was intentional: I was writing an `empty` service and wanted to use the empty listener name. 🤣 

Thanks for fixing!